.github/workflows/trunk.yml

```
@@ -51,8 +51,17 @@ jobs:
           echo "matrix=$matrixStringifiedObject" >> $GITHUB_OUTPUT
         env:
           PROJECT_ROOT: .
+      - name: Get Latest Github Relaase
+        id: get_latest_release
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:34: Double quote to prevent globbing and word splitting

(shellcheck)